doc/source/getting_started/qtcrashcourse.rst

```diff
@@ -91,6 +91,142 @@ http://qt-project.org/doc/qt-4.8/graphicsview.html#the-graphics-view-coordinate-
+
+        def mousePressEvent(self, event):
+            # This method checks if the left mouse button was pressed on the widget
+            # and prints the position of the click.
+            if event.button() == Qt.MouseButton.LeftButton:
+                print("Left mouse button pressed at:", event.position())
+
+    # Initialize the QApplication
+    app = QApplication([])
+    window = MainWindow()
+    window.show()
+    # Start the event loop
+    app.exec()
+
+
+
+This code snippet demonstrates initializing a basic PyQt6 application that responds 
+to a left mouse button click, illustrating the practical application of handling 
+mouse events in a PyQtGraph environment.
+
+Keyboard Events
+^^^^^^^^^^^^^^^
+Keyboard inputs are similarly handled by overriding 
+`keyPressEvent`_, and `keyReleaseEvent`_, allowing applications to 
+react to various keystrokes and facilitate shortcuts and other interactions.
+
+.. _keyPressEvent: https://doc.qt.io/qt-6/qwidget.html#keyPressEvent
+.. _keyReleaseEvent: https://doc.qt.io/qt-6/qwidget.html#keyReleaseEvent
+
+
+Integration with PyQtGraph
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Example: Handling Keyboard Inputs:: 
+
+    from PyQt6.QtWidgets import QApplication, QMainWindow
+    from PyQt6.QtCore import Qt
+
+    # Should work with PyQt5 / PySide2 / PySide6 as well
+
+    class MainWindow(QMainWindow):
+        def __init__(self):
+            super().__init__()
+            # Sets the title of the window
+            self.setWindowTitle('Keyboard Input Tracker')
+            # Sets the position and size of the window
+            self.setGeometry(100, 100, 400, 300)
+
+        def keyPressEvent(self, event):
+            # Checks if a specific key was pressed
+            if event.key() == Qt.Key.Key_Escape:
+                print("Escape key was pressed.")
+            elif event.key() == Qt.Key.Key_Space:
+                print("Space bar was pressed.")
+            else:
+            # The 'event.text()' method retrieves the character or characters 
+            # associated with the key press, and then prints it to the console.
+                print(f"Key pressed: {event.text()}")
+
+    # Initialize the QApplication
+    app = QApplication([])
+    window = MainWindow()
+    window.show()
+    # Start the event loop
+    app.exec()
+
+
+Event Propagation
+^^^^^^^^^^^^^^^^^
+In PyQt, when an event is not handled by a widget, or the widget explicitly decides 
+against handling it, the event is propagated to its parent widget. This process, 
+commonly referred to as "bubbling," continues upward through the nested widgets until 
+the event is either handled or reaches the main window.
+
+It is facilitated by methods such as 
+`accept`_, and `ignore`_, which allow developers to 
+exert precise control over the event flow.
+
+.. _accept: https://doc.qt.io/qt-6/qevent.html#accept
+.. _ignore: https://doc.qt.io/qt-6/qevent.html#ignore
+
+Example: Custom Event Handling ::
```

Comment:
This block is  still long enough that's being clipped in the rendered output:

<img width="770" alt="image" src="https://github.com/pyqtgraph/pyqtgraph/assets/646398/4d38aa7a-c6ba-4b4d-b287-a58308138d43">